tanner/tests/test_config.py

```diff
@@ -73,8 +73,8 @@ def test_get_when_file_dont_exists(self):
             'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
             'EMULATORS': {'root_dir': '/opt/tanner'},
             'EMULATOR_ENABLED': {'sqli': True, 'rfi': True, 'lfi': True, 'xss': True, 'cmd_exec': True},
-            'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
-                     'password': 'user_pass'},
+            'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': '127.0.0.1', 'user': 'root',
```

Comment:
Isn't it possible to leave it as it is and change the values of `host` and `password` in the `setUp` method?

Reply:
Yeah! we can hardcode the credentials in setup directly if we can?

Reply:
I don't think we can test db helpers without changing that. We can only do this for emulators tests.

Reply:
Look [here](https://stackoverflow.com/questions/30945870/how-to-mock-patch-a-class-imported-in-another-module), you can patch the `TannerConfig` class, see if it works.